packages/metric/README.md

```diff
@@ -73,6 +72,62 @@ export class TestClassController {
 
 export class AppModule {}
 ```
+### MetricModule
+The MetricModule provides functionality for working with metrics in your application. 
+It adds the MetricService to the application using the IMetricService token.
+
+```typescript
+MetricModule.register({
+  graphiteApiEndpoint: string,
+  metricsConfig: { prefix: string, interval: number },
+})
+```
+- graphiteApiEndpoint: The URL for connecting to the Graphite API.
+- metricsConfig: Metrics configuration, including the metric prefix and interval.``
```

Comment:
`metricsConfig.prefix` - prefix for metric entry name

`metricsConfig.interval` - period of metric sending in ms